editor/editor.tsx

```diff
@@ -247,19 +249,24 @@ function Editor(props: any) {
 
     // publish event listeners to Gosling.js
     useEffect(() => {
-        // if (gosRef.current) {
-        //    gosRef.current.api.subscribe('rawdata', (type: string, data: RawDataEventData) => {
-        //        console.log('rawdata', data);
-        //    });
-        //     gosRef.current.api.subscribe('click', (type: string, data: CommonEventData) => {
-        //         gosRef.current.api.zoomTo('bam-1', `chr${data.data.chr1}:${data.data.start1}-${data.data.end1}`, 2000);
-        //         gosRef.current.api.zoomTo('bam-2', `chr${data.data.chr2}:${data.data.start2}-${data.data.end2}`, 2000);
-        //         console.log('click', data.data);
-        //     });
-        // }
-        // return () => {
-        //     gosRef.current.api.unsubscribe('rawdata');
-        // }
+        if (gosRef.current) {
+            // gosRef.current.api.subscribe('rawdata', (type: string, data: RawDataEventData) => {
```

Comment:
Planning to add a table in a separate PR to show hovering/clicking event data in the Editor.